modules/data-entry/src/main/frontend/src/questionnaireEditor/Question.json

```
@@ -435,7 +435,7 @@
       },
       "time": {
         "minAnswers": {"0":{}, "1":{}},
-        "dateFormat": {"HH:mm": {}, "mm:ss": {}, "HH:mm:ss":{}},
+        "dateFormat": {"hh:mm a": {}, "HH:mm": {}, "hh:mm:ss a": {}, "HH:mm:ss":{}, "mm:ss": {}},
```

Now that the code default is different, should the designer default also be different?


```suggestion
        "dateFormat": {"HH:mm": {}, "hh:mm a": {}, "HH:mm:ss":{}, "hh:mm:ss a": {}, "mm:ss": {}},
```

---

ok